src/test/resources/Data.json

```diff
@@ -2,6 +2,9 @@
   "url": "https://www.flipkart.com",
   "select_shoe": "2,3,4",
   "paymentPageHeader":"Payments",
-  "expectedQuestionsOnPaymentPage":"14"
+  "expectedQuestionsOnPaymentPage":"14",
+  "policyElementsWhichNeedToClick": "Return Policy,Terms Of Use,Security,Privacy,Sitemap,EPR Compliance",
+  "policyHeaders": "Returns Policy,Flipkart Terms of Use,Safe and Secure Shopping,Privacy Policy,Sitemap,FLIPKART INDIA PRIVATE LIMITED E-WASTE RECYCLING POLICY",
+  "y_axis": "1302,40"
```

Comment:
Wouldn't this change as per browser window size?
If yes then I think we need to dynamically get that size